src/Main.java

```diff
@@ -1,5 +1,81 @@
+import java.time.LocalDate;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+
+        System.out.println("Task 1: ");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введите год: ");
+        checkYearAndPrint(scanner.nextInt());
+
+        System.out.println("Task 2: ");
+        System.out.println("Введите, какая у вас операционная система (0 — iOS, 1 — Android): ");
+        int clientOS = scanner.nextInt();
+        System.out.println("Введите год выпуска вашей модели: ");
+        int yearOfReleaseDevice = scanner.nextInt();
+        checkOSAndYear(clientOS, yearOfReleaseDevice);
+
+        System.out.println("Task 3: ");
+        byte deliveryDistance = 95;
+        checkDistance(deliveryDistance);
+        byte daysDelivery = checkDistance(deliveryDistance);
+        if (daysDelivery > 0) {
+            System.out.println("Потребуется дней: " + daysDelivery);
+        } else {
+            System.out.println("Свыше 100км доставки нет.");
+        }
     }
-}
\ No newline at end of file
+
+    public static void checkYearAndPrint(int year) {
+        if (year > 1584) {
+            if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)) {
+                System.out.println(year + " год является високосным.");
+            } else {
+                System.out.println(year + " год не является високосным.");
+            }
+        } else {
+            System.out.println(year + " год не является високосным.");
+        }
+    }
+
+    public static void checkOSAndYear(int number, int Year) {
```

Comment:
```suggestion
    public static void checkOSAndYear(int number, int year) {
```
названия переменных надо писать с маленькой буквы